docs/whirlpool/docs/03-Whirlpools SDK/02-Whirlpool Management/01-Create Pool.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_label: Create Liquidity Pools
+---
+
+# Creating Liquidity Pools on Orca
+
+Creating liquidity pools on Orca is an essential step for launching your token and enabling trading. In this guide, we'll explore two types of liquidity pools available in the Orca ecosystem, **Splash Pools** and **Concentrated Liquidity Pools**, and help you understand how to create them, their differences, and which one best suits your needs.
+
+## 1. Introduction to Pool Types
+
+### Overview
+
+Liquidity pools are a foundational concept in DeFi, enabling users to trade tokens without relying on traditional order books. On Orca, liquidity pools provide the means for traders to swap between two tokens, while liquidity providers earn fees by supplying the tokens to the pool.
+
+### Splash Pools vs. Concentrated Liquidity Pools
+
+- **Splash Pools**: Splash Pools are the simplest type of liquidity pool. They are ideal for those looking to launch a new token with minimal parameters. You only need to provide the mint addresses of the two tokens and set the initial price. Splash Pools offer an easy entry point into liquidity provision, making them especially appealing for community-driven projects like memecoins. These projects often prioritize community engagement over technical complexity, and Splash Pools provide a straightforward way to get started.
+
+- **Concentrated Liquidity Pools:** Concentrated Liquidity Pools are more advanced and allow liquidity providers to concentrate their liquidity within specific price ranges. This results in higher capital efficiency but requires a deeper understanding of how to manage liquidity. Concentrated Liquidity Pools are better suited for experienced users who want greater control over their liquidity. However, they also come with the risk of divergence loss, where the value of your position may decrease compared to simply holding the tokens individually.
```

Comment:
Not 100% sure on this but no divergence loss on splash pools or just very little that it is negligible?

Reply:
Divergence loss is only mentioned in relation to Concentrated Liquidity Pools.

Reply:
Yeah but is there really no divergence loss on splash pools (or full range positions in CL pools)?

Reply:
Ahh, got it. I think I misread your first comment and interpreted it as a statement - that there's no divergence loss for splash pools. But indeed, you could have divergence loss in full range positions. I will update!

Reply:
I removed that bit from this doc and updated the explainer on divergence loss in the doc Open Position.md, where I think it's more appropriate.